src/parser/state_machine/syntax/comment.rs

```diff
@@ -42,16 +42,16 @@ define_state_group!(comment_states_group = {
     }
 
     comment_less_than_sign_state {
```

Comment:
https://html.spec.whatwg.org/multipage/parsing.html#comment-less-than-sign-state

both changed state transitions have "append the current input character to the comment token's data" in their actions